x-pack/plugins/beats_management/public/components/navigation/breadcrumb/breadcrumb.tsx

```diff
@@ -53,7 +53,7 @@ export const Breadcrumb: React.FC<BreadcrumbProps> = ({ title, path, parentBread
     {context => (
       <BreadcrumbManager
         text={title}
-        href={`#${BASE_PATH}${path}`}
+        href={path ? `#${BASE_PATH}${path}` : `#${BASE_PATH}`}
```

Comment:
This was broken before and would result in URLs like `#/management/beats_managementundefined`